src/views/integrations/event-tickets/emails/template-parts/header/head/tec-styles.php

```diff
@@ -170,4 +170,8 @@
 	td.tec-tickets__email-table-content-event-venue-website-container {
 		padding: 0;
 	}
+	
+	h3.tec-tickets__email-table-content-event-title a {
+		text-decoration: none;
```

Comment:
Should we also set the color so that it's not defaulting to the document default? This could probably be something when viewing the email in some clients.

Reply:
Yes, but there should be a default text color coming from Event Tickets for all regular texts, right? Do we have something like that in ET?